search/__init__.py

```diff
@@ -20,8 +20,8 @@ def get_sqs():
     # Add hard-coded facets.
     for facet in ("course", "run", "resource_type"):
         sqs = sqs.facet(facet)
-    # Add dynamic facets (from taxonomy). Values with spaces do not work,
-    # so use the slug.
-    for slug in Vocabulary.objects.all().values_list("slug", flat=True):
-        sqs = sqs.facet(slug)
+    # Add dynamic facets (from taxonomy). Certain characters cause problems,
+    # so use the primary key.
+    for v_id in Vocabulary.objects.all().values_list("id", flat=True):
+        sqs = sqs.facet(v_id)
```

Comment:
How does the SearchQuerySet know that `v_id` is a vocabulary id and not a term id or other number?